flow/probes/ebpf.go

```diff
@@ -310,7 +309,8 @@ func (p *EBPFProbe) run() {
 					flowChan <- fl
 
 				}
-
+				//delete every entry after we read to reduce size.
+				p.module.DeleteElement(p.fmap, unsafe.Pointer(&key))
```

Comment:
can you move this after this line
https://github.com/skydive-project/skydive/pull/1599/files#diff-733c12abf4ac0cf47dc4a9dddd3b9c40R278

Reply:
yeah agreed, the value has copied locally already